### ISSUE TITLE
Publish to npm as a global binary package

### ISSUE THREAD

**Issue by Glavin001** [sookool99/Hacka-data#1]
### Expected installation

```
npm install -g hacka-data
```
### Expected usage

```
hacka-data "Location" [categories]
```

**Comment by shawnmacburnie** [comment]
Fixes in 3b0deff922eab21b5d7dd68f6ac89f829df8b437 but forgot to reference.

Issue resolved.

**Comment by Glavin001** [comment]
> but forgot to reference.

I forgive you, this once. :wink:
